ts/IterableInboxMessageList.tsx

```diff
@@ -82,11 +85,11 @@ const IterableInboxMessageList = ({
 
    const inboxSessionItemsChanged = useCallback((
       (info: {viewableItems: Array<ViewToken>, changed: Array<ViewToken>}) => {
-         // logCurrentVisibleRows(info)
+         logCurrentVisibleRows(info)
```

Comment:
Use Iterablelogger inside logCurrentVisibleRows and use verbose mode so that its not executed when app is published.

Reply:
Oh, I wrote that method, I should note it was ONLY meant for debugging, not for production usage. So we should delete it before release.

Reply:
@evangreer91 feel free to just get rid of that function.

Reply:
I will delete that function and the function call. Thanks.